packages/system/redis-operator/Makefile

```diff
@@ -1,18 +1,11 @@
 NAME=redis-operator
-NAMESPACE=cozy-redis-operator
+NAMESPACE=cozy-$(NAME)
 
-show:
-	helm template --dry-run=server -n $(NAMESPACE) $(NAME) .
-
-apply:
-	helm upgrade -i -n $(NAMESPACE) $(NAME) .
-
-diff:
-	helm diff upgrade --allow-unreleased --normalize-manifests -n $(NAMESPACE) $(NAME) .
+include ../../hack/package-system.mk
 
 update:
 	rm -rf charts
-	helm repo add redis-operator https://spotahome.github.io/redis-operator
-	helm repo update redis-operator
-	helm pull redis-operator/redis-operator --untar --untardir charts
-	sed -i '/{{/d' charts/redis-operator/crds/databases.spotahome.com_redisfailovers.yaml
+	helm repo add $(NAME) https://spotahome.github.io/$(NAME)
+	helm repo update $(NAME)
+	helm pull $(NAME)/$(NAME) --untar --untardir charts
+	sed -i '/{{/d' charts/$(NAME)/crds/databases.spotahome.com_redisfailovers.yaml
```

Comment:
I would rather keep update target without using variables, because commands here targets to concrete repositories, they are node depended on release name or namespace.